lambdas/service/app.py

```diff
@@ -911,7 +949,14 @@ def repository_search(entity_type: str, item_id: str):
         upper and lower bounds, and multiple pairs are combined using "and"
         logic. For example, `{"donorCount": {"within": [[1,5], [5,10]]}}`
         selects entities whose donor organism count falls within both
-        ranges, i.e., is exactly 5.''' + f'''
+        ranges, i.e., is exactly 5.
+
+        The organismAge facet is special in that it contains two property keys,
+        value and unit. For example,
+        `{"organismAge": {"is": [{"value": "20", "unit": "year"}]}}`. Both keys
+        are required.
```

Comment:
Rewrap the entire paragraph. I really shouldn't have to point out trivial things like this. It's unprofessional to rely on me to catch this.